lib/cisco_node_utils/bgp_neighbor_af.rb

```diff
@@ -308,9 +303,7 @@ def default_disable_peer_as_check
     # -----------------------
     # <state> filter-list <str> in
     def filter_list_in
-      str = config_get('bgp_neighbor_af', 'filter_list_in', @get_args)
-      return default_filter_list_in if str.nil?
-      str.shift.strip
+      config_get('bgp_neighbor_af', 'filter_list_in', @get_args)
```

Comment:
Is the `strip` method not needed?  Comment applies to the other string properties.